docs/v8-migration-api.md

```diff
@@ -110,6 +110,74 @@ Check out the [v8-migration guide](v8-migration.md) for other user-level changes
 
   __Migration__: If you have a customized plugin accessing `typeParameter` of a `TSMappedType` node, use `node.key` and `node.constraint` in Babel 8.
 
+![medium](https://img.shields.io/badge/risk%20of%20breakage%3F-medium-yellow.svg)
+
+- Split `TsExpressionWithTypeArguments` into `TSClassImplements` and `TSInterfaceHeritage` ([#16731](https://github.com/babel/babel/pull/16731)).
+
+  This is to align the AST for TS nodes with `@typescript-eslint`.
+
+  ```ts
+  // Example input
+  class C implements X<T> {}
+  interface I extends X<T> {}
+
+  // AST in Babel 7
+  {
+    type: "ClassDeclaration",
+    id: Identifier("C"),
+    implements: [
+      {
+        type: "TsExpressionWithTypeArguments",
```

Comment:
Since we have also removed the `t.tsExpressionWithTypeArguments` builder in Babel 8, could you also add a migration note within the `@babel/types` section?

Reply:
Maybe just mention in the AST changes that the corresponding builders and validators are removed as well? Since we also removed the validators in `types` and `traverse`.